krun.py

```diff
@@ -6,16 +6,16 @@
 usage: runner.py <config_file.krun>
 """
 
-import os, sys, json, time
+import argparse
+import os, sys, time
```

Comment:
I've been loosely trying to keep the krun source PEP8 (it's the first softdev project to do so). PEP8 (stupidly) says one import per line. I know the old line was wrong in the regard, but since we are changing it, we may as well comply.